deploy/roles/shared/handlers/main.yml

```diff
@@ -4,7 +4,7 @@
   when: (ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux') and ansible_distribution_version == 6
 
 - include_tasks: rhel7.yml
-  when: (ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux') and ansible_distribution_version == 7
+  when: (ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux') and ansible_distribution_version == "7"
```

Comment:
Normally we don't include the distribution check, although I suppose this is technically a more robust approach.  For now perhaps we should just change this to check the version in order to be consistent with other roles.

Reply:
Probably. Maybe common or shared should do a check that it's RHEL or CentOS, and give a warning (or even error) if it's not.

Reply:
Dropped the `ansible_distribution` clauses, changed `ansible_distribution_version` to `ansible_distribution_major_version`, and fixed the unquoted 6 a couple lines above.